src/app/common/hooks/analytics/use-analytics.ts

```diff
@@ -24,6 +24,15 @@ function isHiroApiUrl(url: string) {
   return /^https:\/\/.*\.stacks.co/.test(url);
 }
 
+let flow: undefined | string = undefined;
+export function setFlow(flowName: string) {
+  flow = flowName;
+}
+let flowInitiatorUrl: undefined | string = undefined;
+export function setFlowInitiatorUrl(url: string) {
+  flowInitiatorUrl = url;
+}
+
```

Comment:
It isn't necessary to set the state here, as it's already set in the `useDefaultRequestParams` hook. Also, I'd try and stay a way from maintaining state this way, unless really needed, as it's working outside the realms of react.

```
const { flow, origin } = useDefaultRequestParams();
```

Also, for security reasons, I'd recommend we just log the `origin` url, rather than the full path, which is already there.